docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,108 @@
+
+- **Security**: IAM solutions provide advanced security features such as MFA, RBCA, and audit trails, which can help to improve the security of your Grafana installation.
+
+- **SSO**: Properly setting up Grafana with your current IAM solution enables users to access Grafana with the same credentials they use for other applications.
+
+- **Scalability**: Adding, updating, or removing users from your user database is immediately reflected in Grafana.
+
+In order to plan an integration with Grafana, assess your organization's current needs, requirements and any existing IAM solutions being used. This should include which set of roles and permissions will be match to each type of user and which groups of users share the same set of permissions and access to shared resources.
+
+After considering needs, chosen IAM solution and taking into consideration the security requirements, the last step is to test it out thoroughly before deploying it to a production environment.
+
+## Internal vs external users
+
+As a first step, determine where your Grafana users are located. Are the users located within your organization, or are they outside your organization?
+
+If the users are within your organization, Grafana might be able to integrate with those users through an identify provider.
+
+If the users are outside your organization, you must provide anonymous access to Grafana, which is not enabled by default.
+
+For information about enabling anonymous access, refer to the [documentation](../../setup-grafana/configure-security/configure-authentication/_index.md#anonymous-authentication)
+
+## Ways to organize users
+
+1. **Security**: Different groups of users or customers should only have access to their intended resources.
+1. **Simplicity**: Reduce scope of dashboards and resources available.
+1. **Cost attribution**: Track and bill costs to their customers, departments, or divisions.
+1. **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana Teams
+
+It makes sense to organize users in [Teams] in order to assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team. A user can belong to multiple teams.
+
+A user can be a Member or an Administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team Administrators can add members to a team and update its settings, such as the team name, team member’s team roles, UI preferences, and the default dashboard to be displayed upon login for the team members.
+
+Teams is a perfect solution to work with a small subset of users. Teams can share resources among other teams.
+
+### Users in Grafana Organizations
```

Comment:
As with the teams, there is a bunch of org content in the grafana docs. Maybe you can add links?